src/Contract/Scripts.purs

```diff
@@ -15,6 +15,11 @@ module Contract.Scripts
   , module TypesScripts
   ) where
 
+import Aeson (class DecodeAeson)
+import Address
+  ( enterpriseAddressMintingPolicyHash
+  , enterpriseAddressScriptHash
+  )
```

Comment:
I think we can remove this import completely, this has been refactored into
```
import Contract.Address
  ( enterpriseAddressScriptHash
  , enterpriseAddressStakeValidatorHash
  , enterpriseAddressValidatorHash
  , scriptHashAddress -- Directly uses Plutus `Address`
  , toValidatorHash -- Directly uses Plutus `Address`
  , typedValidatorBaseAddress
  , typedValidatorEnterpriseAddress
  , validatorHashBaseAddress
  , validatorHashEnterpriseAddress
  ) as Address
```
below and I don't think we should include `enterpriseAddressMintingPolicyHash`